extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/binder/MongoParserVisitor.java

```diff
@@ -100,4 +100,13 @@ public String visitPathExpression(HqlParser.PathExpressionContext ctx) {
         // this is the name of the field, we apply replacement and escape with '
         return "'" + replacementMap.getOrDefault(ctx.getText(), ctx.getText()) + "'";
     }
+
+    @Override
+    public String visitInPredicate(HqlParser.InPredicateContext ctx) {
+        StringBuilder sb = new StringBuilder(ctx.expression().accept(this))
+                .append(":{$in:[")
```

Comment:
All operators are escaped with single quotes.
I know this is not mandatory but I prefere to keep it this way

Reply:
Done.